package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "alsatian",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

Comment:
Can you roll back this version change please as this happens automatically on publish :)

Reply:
:+1: